testnet/stacks-node/src/mockamoto.rs

```diff
@@ -339,18 +766,20 @@ impl MockamotoNode {
             (tip_info.stacks_block_height, tip_info.burn_header_height)
         };
 
-        let miner_nonce = 2 * parent_chain_length;
+        info!("Mining block"; "parent_chain_length" => parent_chain_length, "chain_tip_bh" => %chain_tip_bh, "chain_tip_ch" => %chain_tip_ch);
+        let miner_nonce = 3 * parent_chain_length;
```

Comment:
Would it be less brittle to just get the miner's account from the Stacks chain tip?

Reply:
Definitely -- updated in 05274b6.